### ISSUE TITLE
Add on request? Slider options

### ISSUE THREAD

**Issue by ghost** [evertiro/historical-redux2#76]
I know this shouldn't be core functionality, which is why I'm requesting it as an add-on. Couldn't think of any other way to ask.

So as to not reinvent the wheel, does anyone have, or is willing to share their, slider option fields? I'm thinking (or am hopeful) someone's done this with the FlexSlider2 Slider already.

Thanks!

**Comment by psaikali** [comment]
What kind of slider? Image upload+text?
If you're talking about a slider that lets you move a handle on an
horizontal bar to define a value between MIN and MAX, I have a field for
that.

**Comment by ghost** [comment]
Hmm.... I was thinking options for the user to define their own image URL (or select from media), slider size, transition type and transition speed. Does that sound like what you have?

**Comment by psaikali** [comment]
Not at all :)

2013/2/3 availium notifications@github.com

> Hmm.... I was thinking options for the user to define their own image URL
> (or select from media), slider size, transition type and transition speed.
> Does that sound like what you have?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/ghost1227/Redux-Framework/issues/76#issuecomment-13048733.

## 

Pierre Saïkali
psaikali@gmail.com

**Comment by ghost** [comment]
Thinking about this more thoroughly, I could use the radio options and drop down options fields to fill the form. I think the hardest part I'm having is the multiple image insertion for the slider. Wondering if there's a way (sort of like what @ghost1227 is working on for the social media fields) where you can have 1 field for 1 upload, then click a plus sign to add a new field and so on - where the field is the uploaded photo file and/or URL of a remote photo. 

So I guess the "add on" idea is to have a multiple upload field with a dynamic set of uploads (user defined by pressing +)

Any thoughts? I'm certainly no expert coder!

**Comment by ghost** [comment]
Anyone have any ideas on creating a multi upload field? I've been trying my hand at it the last couple of days during my spare time, and don't seem to be getting very far with it actually working. I think my failures lie within the jquery since that's not my strong suit.

**Comment by hittheroadjack** [comment]
Have done the basic work on setting up a rudimentary slider based on FlexSlider2 (needs some CSS). For a proof of concept, I solved the upload issue by simply showing images attached to a post by entering a post title into a text input field.
As I understand from this thread, there should also be a media uploader for multiple files. For me, that makes good sense using the wp 3.5 media editor creating a gallery for this and the option will hold the gallery id which is referenced or generate a gallery shortcode in a text field for copy and paste - also with regards to exporting the final result to the front end, later on. I will dig into that the next few days. 
What else should be adjustable?
dimensions? transitions? easing? css styling?
![flexslide1](https://f.cloud.github.com/assets/3947889/305982/3a3251e4-966a-11e2-9f6b-43081003cdf2.png)

**Comment by psaikali** [comment]
These are my own two cents of course, but I'm wondering why you're wishing to include this inside Redux / Theme Options. 
I think that a site slideshow/slider is related to the site content itself, not the theme configuration. I'd rather create a custom "Slide" post type with a custom image field, text, link, title, etc... rather than implementing that into my theme. And then load the X latest slide posts on the homepage (or anywhere else).

**Comment by hittheroadjack** [comment]
For me, there are two reasons for it:
- you can add a preview (ad?) of your other works into your theme/plugin admin page saving space
- you can use it as a preview what image is contained in what post
-  Could also think about exporting it as a whole to the front end as a short code for Flexslider2 in your posts or       adding a media button to the page editor to choose from (in theme/plugin admin) pre-defined galleries/sliders.
  Think other guys will even have more interesting reasons for doing so.

**Comment by hittheroadjack** [comment]
Need a little help!
Does anyone know how to run actions/filters inside a custom field php?
I tried "add_action('admin_init', array($this, 'function_name');" but no fun . The function is never run.
I also tried putting the add_action call inside the _construct part - still no success. 
I need to hook in early before any output actions for a Post-Redirect-Get pattern.
Appreciate any help on this.
